score/liquid_icx/tests/test_integrate_liquid_icx.py

```diff
@@ -100,6 +84,17 @@ def test_2_join_cap(self):
                 break
         self.assertEqual(1, len(self._get_wallets()))
         owner = self._get_wallet(self._wallet.get_address())
-        self.assertEqual(7, len(owner["join_values"]))
-        self.assertEqual(hex(7 * 30 * 10 ** 18), owner["locked"])
+        self.assertEqual(6, len(owner["join_values"]))
+        self.assertEqual(hex(6 * 30 * 10 ** 18), owner["locked"])
+
+    def test_3_join_with_SCORE(self):
```

Comment:
I'd love to see comments like in `test_integrate_liquid_icx_with_fake_SCORE.py`, because it's really hard to grasp the meaning of these test cases